src/views/Layout.tsx

```diff
@@ -25,25 +18,28 @@ export function Layout() {
 				</header>
 				<main className="Layout-main">
 					<Outlet />
+					{!!user ? null : <SignInButton />}
```

Comment:
Rule of thumb: If you ever see `null` in a ternary, you don't need a ternary

```suggestion
					{user ?? <SignInButton />}
```

Reply:
This is a good rule of thumb to know and to remember that the `??` exists. I definitely looked at is was like 🤔 but I couldn't recall the other option or find the words for the google search!

Reply:
when trying the suggestion it causes a type error because it seems like user isn't a react node.

would something other than this work?

```
{user ? <></> : <SignInButton />}

```

with this it works but it seems odd to just have a fragment to get it to work.